src/platform/plugins/shared/discover/public/plugin.tsx

```diff
@@ -451,6 +451,11 @@ export class DiscoverPlugin
         discoverServices,
       });
     });
+
+    plugins.embeddable.registerTransforms(SEARCH_EMBEDDABLE_TYPE, async () => {
+      const { searchEmbeddableTransforms } = await import('../common/embeddable');
```

Comment:
```suggestion
      const { searchEmbeddableTransforms } = await getEmbeddableServices();
```
Nit: this could probably be grouped with the `embeddable_services` bundle to limit async module imports: https://github.com/elastic/kibana/blob/9c1835613641710cc19b7af73bf1c8d3f9c7c5e5/src/platform/plugins/shared/discover/public/plugin.tsx#L458